scripts/run-integration-tests.sh

```diff
@@ -30,4 +30,9 @@ for ((i = "$agent_number"; i <= "${#test_dirs[@]}"; )); do
 done
 
 printf "\nRunning the following test suites:\n\n%s\n\nStarting tests...\n\n" "$(echo "${dirs[@]}" | tr -s ' ' '\n')"
-ginkgo -keepGoing --slowSpecThreshold 60 "${dirs[@]}"
+
+if [ -z "${PKCS11_LIB-}" ] || [ -z "${PKCS11_LABEL-}" ] || [ -z "${PKCS11_PIN-}" ]; then
```

Comment:
Can we avoid this? The suite can be skipped within the integration test if these aren't set.